simpleform/src/main/java/com/telran/simpleform/controller/YourPetController.java

```diff
@@ -0,0 +1,39 @@
+package com.telran.simpleform.controller;
+
+
+import com.telran.simpleform.controller.dto.YourPetDto;
+import com.telran.simpleform.repository.PetRepository;
+import com.telran.simpleform.repository.UserRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/apiPet")
```

Comment:
@den030 We can do just @RequestMapping("/api/pet") and in the case we don't need map controller methods to "pet"